### PR TITLE
FAQs on Watch Folder Settings

### LINKED ISSUE #191: FAQ Section on Mail processing / Watch folder settings

### PR DESCRIPTION
**Pull request by ghost**
I answered the questions from #191, which arise when you look at the folder watch settings in the app.

closes #191

### CODE REVIEW

**Comment by ghost** [comment]
@hpk42 does this PR answer the questions from #191? https://staging.delta.chat/259/en/help#multiclient

File: en/help.md [hunk 1]
```diff
@@ -345,6 +345,49 @@ capable) devices, you have to sync their encryption capabilities:
...
+
+The default setting for copy-to-self is "no".
+
+### Why can I choose to watch the "Sent" folder?
+
+The only reason one wants to watch the Sent folder is if you are using another
+mail program (like Thunderbird) next to your Delta Chat app, and want your MUA
+to participate in chat conversations.
+
+Instead we recommend to use the Delta Chat Desktop Client; you can download it
```
Comment (hpk42) [inline en/help.md]:
```suggestion
However, we recommend to use the Delta Chat Desktop Client; you can download it
```

File: en/help.md [hunk 2]
```diff
@@ -345,6 +345,49 @@ capable) devices, you have to sync their encryption capabilities:
...
+on [get.delta.chat](https://get.delta.chat).
```
Comment (hpk42) [inline en/help.md]:
```suggestion
on [get.delta.chat](https://get.delta.chat). The option to watch the "Sent" folder might go away in the future. It was introduced at a time where there was no Delta Chat Desktop client available on all platforms. 
```

File: en/help.md [hunk 3]
```diff
@@ -345,6 +345,49 @@ capable) devices, you have to sync their encryption capabilities:
...
+
+### Why can I choose not to watch the DeltaChat folder?
+
+Some people use Delta Chat as a regular email client, and want to use the Inbox
+folder for their mail, instead of the DeltaChat folder. If you disable "Watch
+DeltaChat folder", you should also disable "move chat messages to DeltaChat".
+Otherwise, some things like deleting messages won't work anymore.
```
Comment (hpk42) [inline en/help.md]:
```suggestion
Otherwise, deleting messages or multi-device setups might not work properly.
```

**Review by hpk42 — APPROVED** [review]
thanks, lgtm but i suggest you merge my change suggestions before merging.

**Pull request merged**